### PR TITLE
Adding member_zrx_staked to epoch pool rewards query

### PR DESCRIPTION
**Pull request by lyaunzbe**
In reference to: https://github.com/0xProject/website/issues/366

Basically, we want to be able to see the amount of ZRX staked per each epoch. Currently, staking endpoint `staking/pools/{poolId}` only shows amount of ZRX staked for current epoch and next epoch. Also added back nodemon dev npm script.

### CODE REVIEW

**Review by xianny — COMMENTED** [review]
Looks good to me but I think @sui414 should review as the data and queries are her expertise :)

File: packages/api/package.json [hunk 1]
```diff
@@ -3,10 +3,13 @@
...
     "main": "index.js",
     "license": "MIT",
-    "nohoist": ["**types**"],
+    "nohoist": [
+        "**types**"
+    ],
     "scripts": {
         "build": "yarn tsc -b",
         "clean": "rm -rf lib",
+        "dev": "nodemon -r dotenv/config src/index.ts | pino-pretty",
```
Comment (xianny) [inline packages/api/package.json:12]: yay, thanks!

**Review by xianny — APPROVED** [review]

**Comment by xianny** [comment]
I took a peek at https://github.com/0xProject/website/issues/366 for more context.

Why not add `epochRewards` to the `/pools` response? It seems we would always want to call `/pools` from the front-end on page load, and could cache that response on localhost to use when clicking through to each pool's individual page. Could also include ETH/ZRX prices in the response. 

Highly recommend doing aggregate queries and joins at the SQL layer instead of the API/network layer. Also, this data changes slowly so we can add a cache at the ingress layer for faster page loads.

File: packages/api/src/queries.ts [hunk 2]
```diff
@@ -328,18 +328,21 @@ export const poolEpochRewardsQuery = `
         , e.ending_timestamp
         , e.ending_block_number
         , e.ending_transaction_hash
+        , esps.member_zrx_delegated AS member_zrx_staked
```
Comment (sui414) [inline packages/api/src/queries.ts:331]: COALESCE(esps.member_zrx_delegated, 0.00)

**Review by sui414 — APPROVED** [review]

**Pull request merged**